pkg/apis/stats/v1alpha1/types.go

```diff
@@ -85,6 +85,34 @@ type NetworkPolicyStats struct {
 	TrafficStats TrafficStats `json:"trafficStats,omitempty" protobuf:"bytes,2,opt,name=trafficStats"`
 }
 
+// +genclient
+// +resourceName=multicastgroupmembers
+// +genclient:readonly
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// MulticastGroupMember contains the mapping between Multicast Group and Pods.
+type MulticastGroupMember struct {
```

Comment:
Do we plan to have more information for MulticastGroup that can be exposed by this API?
If no, current API name and group work for me.
If yes, I wonder if we should just use single API for all information specific to MulticastGroup, name it `MulticastGroup`.

Reply:
No, we don't have plan to expose more group-level multicast information.

Reply:
It should be called `MulticastGroupMembers`, but I also feel a more generic name like `MulticastGroup` or `MulticastGroupInfo` is better for future extensions.

Reply:
Updated. 

Changed struct name to MulticastGroup. Trade explicitness for extensibility seems to be a good deal for me.